feat: add delete conversation functionality and enhance conversation … (#81)
Cognigy/Webchat: docs/embedding.md

| No network                            | string | "No network connection"                                                                     |
| Reconnecting                          | string | "Reconnecting..."                                                                           |
| Reconnect                             | string | "Reconnect"                                                                                 |
| conversations_options                 | string | "Conversations Options"                                                                     |
| delete_all_conversations              | string | "Delete all conversations"                                                                  |
| delete_all_conversations_confirmation | string | "You are about to delete all the conversation. This action cannot be undone. Are you sure?" |
| delete_conversation                   | string | "Delete Conversation"                                                                       |

> | delete_conversation_confirmation      | string | "You are about to delete the conversation. This action cannot be undone."                   |

Author: ```suggestion
| delete_conversation_confirmation      | string | "Are you sure you want to delete this conversation? This action cannot be undone"                   |
```
